### PR TITLE
update to use latest NAD 2024 schema

### PR DESCRIPTION
**Pull request by gregbunce**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Python27\\ArcGIS10.7\\python.exe"
```
Comment (steveoh) [inline .vscode/settings.json:2]: This has been unnecessary with new releases of vs code and the python plugin for a while now. Was there a reason you had to add it?

File: MAINTAINERS [hunk 2]
```diff
@@ -1 +1,2 @@
 keatonwalker
```
Comment (steveoh) [inline MAINTAINERS:1]: You should remove him or he'll be added to or reviews

File: etl_nad.py [hunk 3]
```diff
@@ -129,13 +129,16 @@ def translateValues(nadPoints):
...
     with arcpy.da.UpdateCursor(nadPoints,
-                               ['SHAPE@X', 'SHAPE@Y', 'longitude', 'latitude', 'Source'],
+                               ['SHAPE@X', 'SHAPE@Y', 'Longitude', 'Latitude', 'NAD_Source'],
                                spatial_reference=arcpy.SpatialReference(4326)) as cursor:
+        x = 0
         for row in cursor:
             row[2] = row[0]
             row[3] = row[1]
-            row[4] = 'Utah AGRC'
+            row[4] = 'Utah UGRC'
```
Comment (steveoh) [inline etl_nad.py:138]: That's a bit of double Utah Utah no?

Comment (gregbunce) [reply etl_nad.py:138]: yup, good catch i fixed it.  it will be reflected in my next PR (this week)

File: MAINTAINERS (hunk 2 above)
Comment (gregbunce) [reply MAINTAINERS:1]: good call.  Thx (90s style)

File: .vscode/settings.json (hunk 1 above)
Comment (gregbunce) [reply .vscode/settings.json:2]: i can't recall why i added that.  i'll remove it and see what happens.

**Pull request merged**